jobs/rocoto/arch.sh

```diff
@@ -182,7 +182,11 @@ if [ $CDUMP = "gfs" ]; then
 
     #for targrp in gfs_flux gfs_netcdf/nemsio gfs_pgrb2b; do
     if [ ${SAVEFCSTNEMSIO:-"YES"} = "YES" ]; then
-        for targrp in gfs_flux gfs_${format}a gfs_${format}b gfs_pgrb2b; do
+	targrp_list="gfs_flux gfs_${format}b gfs_pgrb2b"
+	if [ $MODE = "cycled" ] ; then
+	    targrp_list="${targrp_list} gfs_${format}a"
+        fi
+        for targrp in $targrp_list; do
             htar -P -cvf $ATARDIR/$CDATE/${targrp}.tar `cat $ARCH_LIST/${targrp}.txt`
```

Comment:
Indentation is rendering as incorrect on GitHub. Check for space/tab mixing.

Reply:
Good catch. I addressed mixed tab/space in that section and went through the rest of the script to replace other tabs with spaces.